lib/get_started/get_started_view.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+
+class GetStartedView extends StatelessWidget {
+  const GetStartedView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xFFf2e9e0),
+      body: CustomPaint(
+        size: const Size(double.infinity, double.infinity),
+        painter: CustomBackgroundPainter(),
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 8.0),
+          child: Stack(
+            children: [
+              Positioned(top: 0,right: -24,
+                  child: Image.asset('assets/images/tarelka-removebg-preview.png',height: 300,width: 300,fit: BoxFit.cover,)),
```

Comment:
![image](https://github.com/user-attachments/assets/e67c74ec-2eac-4e66-b9d1-89c54ffb1bdb)
No images import in commit and on even if they have to be they going to override the text below

Reply:
+ i mentioned that images have padding from right side . They shouldn't have it

Reply:
pubspec was updated with assets